sdk/kms/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@mysten/kms",
+	"version": "0.0.1",
+	"description": "A collection of KMS signers for various cloud providers",
+	"license": "Apache-2.0",
+	"author": "Mysten Labs <build@mystenlabs.com>",
+	"type": "commonjs",
+	"exports": {
+		"./aws": {
+			"import": "./dist/esm/aws/index.js",
+			"require": "./dist/cjs/aws/index.js"
+		}
+	},
+	"sideEffects": false,
+	"files": [
+		"CHANGELOG.md",
+		"LICENSE",
+		"README.md",
+		"aws",
+		"dist",
+		"src"
+	],
+	"scripts": {
+		"clean": "rm -rf tsconfig.tsbuildinfo ./dist",
+		"build": "build-package",
+		"prepublishOnly": "pnpm build",
+		"prettier:check": "prettier -c --ignore-unknown .",
+		"prettier:fix": "prettier -w --ignore-unknown .",
+		"eslint:check": "eslint --max-warnings=0 .",
+		"eslint:fix": "pnpm run eslint:check --fix",
+		"lint": "pnpm run eslint:check && pnpm run prettier:check",
+		"lint:fix": "pnpm run eslint:fix && pnpm run prettier:fix",
+		"test": "vitest"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/mystenlabs/sui.git"
+	},
+	"bugs": {
+		"url": "https://github.com/mystenlabs/sui/issues"
+	},
+	"homepage": "https://github.com/mystenlabs/sui#readme",
+	"devDependencies": {
+		"@mysten/build-scripts": "workspace:*",
+		"@types/node": "^20.14.10",
+		"typescript": "^5.5.3",
+		"vitest": "^2.0.1"
+	},
+	"dependencies": {
+		"@mysten/sui": "workspace:*",
```

Comment:
@hayes-mysten Any thoughts on moving new packages to use peer deps instead of direct deps?

Reply:
Peer deps always scare me, I haven't used them in a while, so not sure how well they work these days.  Its probably the more correct way to do this.

My main concern is that peer deps and changesets don't get along well, and I don't recall exactly what the current state of that is.